torch/csrc/api/include/torch/special.h

```diff
@@ -21,6 +21,22 @@ inline Tensor& gammaln_out(Tensor& result, const Tensor& self) {
   return torch::special_gammaln_out(result, self);
 }
 
+/// Computes the nth derivative of the digamma function on the input.
+/// See https:://pytorch.org/docs/master/special.html#torch.special.polygamma.
+///
+/// Example:
+/// ```
+/// auto t = torch::randn(128, dtype=kDouble);
+/// torch::special::polygamma(2, t);
+/// ```
+inline Tensor polygamma(int64_t n, const Tensor& self) {
+  return torch::special_polygamma(n, self);
+}
+
+inline Tensor& polygamma_out(int64_t n, Tensor& result, const Tensor& self) {
```

Comment:
We should stick to the signature of `polygamma_out` which is `Tensor& polygamma_out(Tensor& result, int64_t n,  const Tensor& self)`